askama_shared/src/escaping.rs

```diff
@@ -55,26 +55,33 @@ fn escapable(b: u8) -> bool {
 }
 
 pub fn escape(s: String) -> String {
-    let mut found = Vec::new();
-    for (i, b) in s.as_bytes().iter().enumerate() {
-        if escapable(*b) {
-            found.push(i);
+    let found: usize = if let Some(c) = s.chars().next() {
```

Comment:
What is the purpose of separately testing the first character, here?

Reply:
Yes, I can change it for a boolean.